nestjs/src/profile/profile.service.ts

```diff
@@ -135,6 +136,55 @@ export class ProfileService {
     }
   }
 
+  public async updateProfileFlat(userId: number, profileInfo: UpdateProfileInfoDto) {
+    const {
+      name,
+      countryName,
+      cityName,
+      educationHistory,
+      discord,
+      englishLevel,
+      aboutMyself,
+      contactsTelegram,
+      contactsPhone,
+      contactsEmail,
+      contactsNotes,
+      contactsSkype,
+      contactsLinkedIn,
+      contactsEpamEmail,
+    } = profileInfo;
+
+    const [firstName, lastName] = name?.split(' ') ?? [];
+
+    const user = await this.userRepository
+      .createQueryBuilder()
```

Comment:
i think you can just use `this.userRepository.update(id, {...});` ?

Reply:
Sure! Thanks